pkg/controller/registry/resolver/operators.go

```diff
@@ -285,13 +302,58 @@ func NewOperatorFromBundle(bundle *api.Bundle, startingCSV string, sourceKey Cat
 		return op, nil
 	}
 
+	// Extract dependencies info
+	depList := []VersionDependency{}
+	for _, v := range bundle.GetDependencies() {
+		switch v.GetType() {
+		case registry.GVKType:
+			gvkDep := registry.GVKDependency{}
+			err := json.Unmarshal([]byte(v.GetValue()), &gvkDep)
+			// Ignore if can't parse JSON string
+			if err != nil {
+				continue
+			}
+			// Add to requiredAPIs list
+			// Set Plural to be empty string
+			required[registry.APIKey{Group: gvkDep.Group, Kind: gvkDep.Kind, Version: gvkDep.Version}] = struct{}{}
```

Comment:
This may be a breaking change, plural is not unused elsewhere.

Reply:
I don't think so given there shouldn't be different APIs with same GVK but different plural forms. We can enable the new resolver and see if any of the existing e2e test cases fail due to this change.